auth/src/main/java/com/firebase/ui/auth/IdpResponse.java

```diff
@@ -25,9 +26,7 @@
  * A container that encapsulates the result of authenticating with an Identity Provider.
  */
 public class IdpResponse implements Parcelable {
-
     private final String mProviderId;
-    @Nullable
```

Comment:
I don't think this should be nullable: we always know the email address and provider id and since having it be nullable caused a crash, I made it @NonNull.